src/locales/en/translation.json

```diff
@@ -1959,7 +1959,12 @@
     "cta": {
       "margin": "Add to Margin",
       "close": "Close"
-    }
+    },
+    "explainers": {
```

Comment:
Please verify these texts with someone as I think they are not 100% correct. I'll suggest some changes but it would be for the best to have it verified by someone else as well.